src/coins.h

```diff
@@ -162,26 +162,19 @@ struct CCoinsCacheEntry
     //! Adding a flag also requires a self reference to the pair that contains
     //! this entry in the CCoinsCache map and a reference to the sentinel of the
```

Comment:
nit: this comment is no longer accurate after d5e3bbf440aa948df8159999fd4eb5275c354b93

We should just remove the part about a self reference, and only mention the sentinel.